### ISSUE TITLE
two ideas to curb metagaming revs

### ISSUE THREAD

**Issue by JoeHammad1844** [space-wizards/space-station-14#22072]
when a headrev flashes somebody and they fall down, its a massive beacon to every witness "HEY I'M A DIRTY REV PLEASE KILL ME" and it's pretty metagamey, my solution is for flashes to just do that by default. it will add some doubt to weather that's a headrev or just an asshole tider with a flash, forceing sec to use mindshields to verify before gibbing them. It will also help with #21197

sunglasses are also meta'd hard in rev rounds, which can be solved by simply making a cheap version of sunglasses that look the same but don't provide flash resistance, have a slightly different description and put them in the clothesmate, like goonstation's tanning sunglasses

**Comment by LankLTE** [comment]
Maybe I’m stupid, but I don’t see how *watching a rev convert someone* and calling them out for it is metagaming? Like that just feels like the fault of the hedrev who is letting there be witnesses to them converting rather than the fault of the witnesses for “metagaming”.

**Comment by asperger-sind** [comment]
Flashes are already borderline overpowered and imo the greatest threat sec can offer, buffing them more is just ridiculous.

**Comment by JoeHammad1844** [comment]
> Flashes are already borderline overpowered and imo the greatest threat sec can offer, buffing them more is just ridiculous.

you can still attack and move if you get flashed and flash protection is easy to get for syndies. nukies even get it for free

**Comment by JoeHammad1844** [comment]
> Maybe I’m stupid, but I don’t see how _watching a rev convert someone_ and calling them out for it is metagaming? Like that just feels like the fault of the hedrev who is letting there be witnesses to them converting rather than the fault of the witnesses for “metagaming”.

you could be in the deepest depths of maints and somebody can still walk in on you, witnesses aren't exclusive to public hallways

**Comment by asperger-sind** [comment]
> you can still attack and move if you get flashed and flash protection is easy to get for syndies.

No and no, you're largely blinded and so your option is to either hit in the general area of whoever is trying to flash you (which they can also do outside of melee range) and the flash slowdown is so great that you wont be able to run away no matter what.

Flash protection is *can* be obtained with relative ease by syndies, see: susbox, but then you get an item that makes you valid and has a clearly visible sprite, and your next best bet is welding masks or killing the lawyer/musician/secoff which there might never be an "easy" way to do.

Trying to openly fight a competent secoff without flashresistance is essentially suicide unless you resort to something like hypopenning them or have some weapon with stupidly low TTK or get them by surprise to slip & cuff them or some shit, making this fight even more one-sided by making flashes drop you is bizarre to me.

**Comment by JoeHammad1844** [comment]
you can also buy a bribery bundle with sunglasses for 4TC, hack into the bartenders room, ask the bartender, and if the PR is merged, buy a 2TC bundle with sunglasses.
officers might not even have their sunglasses on anyway

flashes also dont need to drop you on the first hit, it could take multiple, like on goonstation.

and just to clarify, im only talking about directly hitting somebody with the flash stunning them

**Comment by Patient212** [comment]
Or... maybe just remove the flash sound for head rev flash? This way only actual witnesses will report by actually _seeing_rather than hearing.

**Comment by deltanedas** [comment]
spamming curtains in sci even more op then
it would need a visual light though since silently swinging a flash in a hallway would be a bit sus

**Comment by Hmeister-fake** [comment]
> killing the lawyer/musician/secoff which there might never be an "easy" way to do.

Barkeeps have sunglasses and a lot of them, and usually give them out if you ask nicely enough

**Comment by Titian3** [comment]
I like the idea of the sunglasses, essentially a sunglasses version of budget insulated gloves.

Not so sure on the make everyone fall over as a resolution, I'd rather change the rev conversion fall over to a scream, which is a more common response to getting flashed. Alternatively no sound at all. Both of which could mitigate the meta parts of the conversion mechanic. Feels fair game if you actually see it happen but if you just hear it from two rooms away and relay on radio its a bit shit.
This also generates a few aHelps too where people hear but not see the conversions.

**Comment by deltanedas** [comment]
yeah screaming is a very normal thing to hear so you wouldnt get ;REVS from someone screaming

**Comment by JoeHammad1844** [comment]
we could keep the knockdown but reduce the time on it, how long does it knock people down for at the moment? and compare that to the slipping knockdown
